theme.toml

```diff
@@ -40,6 +40,7 @@
 
 "ui.background" = { bg = "#3b224c" } # midnight
 "ui.linenr" = { fg = "#5a5977" } # comet
+"ui.linenr.selected" = { fg = "#dbbfef" } # lilac
```

Comment:
I think you should just change the default so everyone can benefit.

Reply:
The `selected` variant does have a default in this (`#dbbfef`) ?

Reply:
Yeah I'm not sure what @pickfire means here, the colors here are intended.